### ISSUE TITLE
Update the "Hide unused terms" feature

### ISSUE THREAD

**Issue by stevejburge** [TaxoPress/TaxoPress#2326]
Original issue: https://github.com/TaxoPress/TaxoPress/issues/1815

We don't need to hide terms in the admin area. The goal here is this:
"Redirect tag pages to the homepage when they have less than X posts in them"

As this is a settings, we probably need two extra features:
- A "Save" button
- A display of any current settings

**Comment by stevejburge** [comment]
I'm not able to get this working in testing

**Comment by Deji98** [comment]
oh! I see your point now @stevejburge 
so, the new update should cover this:
1. The hidden terms shouldn't be shown on the front of the site
2. A "Column" in the terms table to show which terms are hidden
3. Redirect the archives( which is what we are already doing on the terms and taxonomies screen)
4. A setting to prevent the hidden terms from being display in "Terms Display", "Terms for Current Post" etc

**Comment by stevejburge** [comment]
@Deji98 Yes, I think we're discovering this approach needs more than we planned.

Let's include a new tab in "Settings" called "Hidden Terms".

There you can enable the feature and choose the minimum number of terms.

If the feature is enabled:
- we hide the terms
- we show the column.
- we redirect the archives
